Composer/packages/client/src/utils/publishStatusPollingUpdater.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import httpClient from './httpUtil';
+
+enum CircuitBreakerStateEnum {
```

Comment:
What is this CircuitBreakerState? Can this be a more descriptive name?

Reply:
rename `PollingStateEnum`